CHANGELOG.md

```diff
@@ -2,11 +2,16 @@
 
 ## Upcoming Release
 
+## v0.19.0
+
 ### Added
 
-- [[#289](https://github.com/rust-vmm/kvm-ioctls/pull/289)]: Drop `x86` and `arm` support.
 - [[#275](https://github.com/rust-vmm/kvm-ioctls/pull/275)]: Introduce `riscv64` ioctls.
 
+### Removed
+
+- [[#289](https://github.com/rust-vmm/kvm-ioctls/pull/289)]: Drop `x86` and `arm` support.
```

Comment:
Let's be absolutely clear there that this is 32 bits :sweat_smile:

Reply:
😂 How true, I'll expand it